mod_wsgi finds /usr/lib/python3.4/config-3.4m/libpython3.4m.a
and links it in. We don't want this to happen, it should use dynamic linking
(especially when we currently don't ship 64-bit version of this static library)

--- mod_wsgi-4.5.17/configure.ac.1	2017-08-07 17:38:33.421758922 +0000
+++ mod_wsgi-4.5.17/configure.ac	2017-08-07 17:39:56.460906766 +0000
@@ -154,8 +154,8 @@
 
 if test "${PYTHONFRAMEWORKDIR}" = "no-framework" -o \
         "${ENABLE_FRAMEWORK}" != "yes"; then
-    LDFLAGS1="-L${PYTHONLIBDIR}"
-    LDFLAGS2="-L${PYTHONCFGDIR}"
+    #LDFLAGS1="-L${PYTHONLIBDIR}"
+    #LDFLAGS2="-L${PYTHONCFGDIR}"
 
     LDLIBS1="-lpython${PYTHON_LDVERSION}"
 
